.gitignore

```diff
@@ -0,0 +1,2 @@
+dist
+node_modules
```

Comment:
When I initially tried running `npm run build` before running `npm i` the error log generated a file called `npm-debug.log`.

I know this is unlikely to occur if things are running smoothly, but might we want to add this to `.gitignore` so in case there is an error it won't get tracked?

Reply:
Good call!